### ISSUE TITLE
[bug

### ISSUE THREAD

**Issue by Hackmastr** [mybb/mybb2#207]

**Comment by euantorano** [comment]
Yep, we haven't done much testing with guest permissions and stuff yet. I completely broke permissions whilst updating the auth package, so I need to fix that first.
